.github/workflows/ci.yaml

```diff
@@ -769,14 +728,6 @@ jobs:
       - name: Set secrets for job
         run: |
           sops -d .secrets.yaml | yq -e '.secrets | to_entries[] | .key + "=" + .value' >> "$GITHUB_ENV"
-      - name: Download artifacts
-        uses: actions/download-artifact@v4
```

Comment:
we don't need this step anymore?

Reply:
We weren't using this step on cron e2e tests, so I presume it's also not needed here.